packages/common/editors/EditorContainer.js

```diff
@@ -51,11 +54,27 @@ class EditorContainer extends React.Component {
   }
 
   componentWillUnmount() {
+    document.removeEventListener('click', this.handleDocumentClick);
     if (!this.changeCommitted && !this.changeCanceled) {
       this.commit({ key: 'Enter' });
     }
   }
 
+  handleDocumentClick = (e) => {
+    const { target } = e;
+    const { container, editor } = this;
+    const isClickedInsideEditorContainer = container !== null && (target === container || container.contains(target));
```

Comment:
Instead of `container !== null && ...` could we use one of the below?

```suggestion
    const isClickedInsideEditorContainer = container && (target === container || container.contains(target));
```

```suggestion
    const isClickedInsideEditorContainer = !!container && (target === container || container.contains(target));
```